tensorflow/python/kernel_tests/cwise_ops_test.py

```diff
@@ -1228,6 +1228,12 @@ def testEmpty(self):
       tf_val = math_ops.polyval(coeffs, x)
       self.assertAllClose(np_val, self.evaluate(tf_val))
 
+  def test_coeffs_raise(self):
+    x = np.random.rand(2, 2).astype(np.float32)
+    coeffs = {}
+    with self.assertRaises(ValueError):
```

Comment:
Before you were also checking the error message. I would recommend checking that here too, using method of `tf.TestCase`

Reply:
I have added that. Can you review it again, please?